.ci/packaging.groovy

```diff
@@ -436,6 +436,7 @@ def runE2ETests(){
   }
   echo 'runE2E will run now in a sync mode to validate packages can be published.'
   runE2E(runTestsSuites: suites,
+         testMatrixFile: '.ci/e2e-tests-beats.yaml',
```

Comment:
I cannot find the file in this branch https://github.com/elastic/beats/tree/8.2/.ci

Reply:
There is one candidate here but it has a `.` in the beginning of the filename https://github.com/elastic/e2e-testing/tree/main/.ci

Reply:
You are right. The original file is a hidden one. Let me do the rework